detox/src/client/Client.js

```diff
@@ -73,16 +73,23 @@ class Client {
 
     try {
       if (this.isConnected) {
-        await this.sendAction(new actions.Cleanup(this._successfulTestRun));
+        this._scheduleAppTermination();
```

Comment:
@d4vidi , actually this decision is a bit arguable — as you see, here I plan to terminate the app if it is unresponsive.

The other option would be just to cancel the cleanup request awaiting and go on. But concerns me, if the app is in broken state, why should we keep it running on the device? Better to kill it, no?

What do you think?

Reply:
After a night of sleep, I think it is more reasonable to let the app hang. Maybe users will find it useful in their local environment. On CI, you can run `detox test ... --cleanup` so that emulator/simulator is terminated after all the tests, so no big sense.